subgraph.template.yaml

```diff
@@ -78,7 +78,7 @@ templates:
           handler: handleRulingAppealConfirmed
         - event: PenaltiesSettled(indexed uint256,indexed uint256,uint256)
           handler: handlePenaltiesSettled
-        - event: RewardSettled(indexed uint256,indexed uint256,address)
+        - event: RewardSettled(indexed uint256,indexed uint256,address,uint256,uint256)
```

Comment:
Wondering on this; what about the events with the old signature on the old DisputeManager? Did it just never emit this event so we don't need to be backwards compatible?

Reply:
Good question, but honestly not sure, we can check it out

Reply:
I could accidentally test this yesterday :)  Because i updated everything but the court contracts that i was deploying to my local devchain and the event just doesn't fire. Didn't experience any graph errors.